src/Mod/Part/App/TopoShapeMapper.cpp

```diff
@@ -1,6 +1,31 @@
-#include "PreCompiled.h"
+// SPDX-License-Identifier: LGPL-2.1-or-later
+/****************************************************************************
+ *                                                                          *
+ *   Copyright (c) 2002 Jürgen Riegel <juergen.riegel@web.de>               *
```

Comment:
Is this the right copyright here?

Reply:
I think so.  All the code here came from the file with this ( TopoShape.cpp ) ; this is just moving to a separate file.  I'm guessing at the right protocol here.  This was intentional, but maybe I'm wrong?

Reply:
It's fine, I just didn't want to mis-attribute if this was a copy-and-paste error.